WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -92,6 +100,13 @@ struct CartView: View {
                             }
                         }
                         .padding(.bottom, floatingControlAreaSize.height)
+                        .background(GeometryReader { geometry in
+                            Color.clear.preference(key: ScrollOffSetPreferenceKey.self,
+                                                   value: geometry.frame(in: coordinateSpace).minY)
```

Comment:
np; I'm not sure if it produces the same result. When doing similar functionality in UIKit, usually I used `origin.y`.

```suggestion
                                                   value: geometry.frame(in: coordinateSpace).origin.y
```

Reply:
Interesting, I didn't really get the difference either by reading the docs, but out friend gpt clarified it a bit further:

> For rectangular frames, this is typically the same as origin.y because the smallest y value in a rectangle is usually at the top edge. `minY` is useful when working with non-rectangular shapes or when the frame might be rotated or transformed, but in most practical cases with standard rectangular frames, minY will give the same result as origin.y.

Since we're working with rectangular shapes here I updated it to `origin.y` on https://github.com/woocommerce/woocommerce-ios/pull/13725/commits/ed868c6a9a78e5fe8251284ae5d217629379250b

Reply:
Interesting, thanks for sharing!